test/e2e/logs_test.go

```diff
@@ -65,7 +65,7 @@ func retrieveLog(nodeName, srcPath, destDir string) error {
 	if err != nil {
 		var exitError *exec.ExitError
 		stderr := ""
-		if errors.As(err, exitError) {
+		if errors.As(err, &exitError) {
```

Comment:
Should this be a separate PR so that we can backport it easily? i.e. fix this file separately. The log collection flake is showing up quite often and I would like us to get to the bottom of that. cc: @sebsoto

Reply:
yes lets break this file's changes out into its own PR, otherwise the process to get this fix in to previous branches will take way too long

Reply:
Sounds good, I'll break them up ASAP